specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -1,14 +1,18 @@
 import "@typespec/rest";
 import "@typespec/http";
 import "@typespec/versioning";
+import "@typespec/openapi3";
```

Comment:
I'd remove this. We don't need to generate OpenAPI v3 for this anyway.

Reply:
I just removed this.